test/IntegrationTests/ModuleTests.cs

```diff
@@ -0,0 +1,98 @@
+// <copyright file="ModuleTests.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+#if NETCOREAPP3_1_OR_GREATER && _WINDOWS
+
+using System;
+using System.Diagnostics;
+using System.Linq;
+using System.Threading.Tasks;
+using IntegrationTests.Helpers;
+using VerifyXunit;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace IntegrationTests;
+
+[UsesVerify]
```

Comment:
Can we execute it on all machines types/framework families?
I suppose that results can be a bit different.

Reply:
Somehow net472 and linux and mac only report single native library. Just needs more research why.

Reply:
https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/pull/1515/commits/138029393eb0b44f4c4a837d3d5e5e8eaec85484